packages/verkle/src/db/checkpoint.ts

```diff
@@ -109,6 +109,7 @@ export class CheckpointDB implements DB {
             opts: { keyEncoding: KeyEncoding.Bytes, valueEncoding: ValueEncoding.Bytes },
           })
         }
+        this._stats.db.writes += 1
```

Comment:
In the trie package checkpoint, we only add writes for `del` operations and `put` operations. Why do we add it here to `commit`?

Reply:
See bellow for full response: https://github.com/ethereumjs/ethereumjs-monorepo/pull/3407#issuecomment-2103267297